@xen-orchestra/lite/src/components/pool/dashboard/PoolDashboardCpuProvisionning.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <UiCard>
+    <UiTitle type="h4" class="title"
+      >{{ $t("cpu-provisioning") }}
+      <!-- TODO: add a tooltip on the icon when available to explain why there is a warning -->
+      <UiIcon v-if="isWarning" :icon="faWarning"
+    /></UiTitle>
+    <div v-if="isReady" class="progress-item" :class="{ warning: isWarning }">
+      <UiProgressBar color="custom" :value="nVCpuInUse" :max-value="maxValue" />
+      <UiUnitProgressBar :max-value="maxValue" />
+      <UiLegendProgressBar>
+        <template #label>{{ $t("vcpus") }}</template>
+        <template #value>{{ value }}%</template>
+      </UiLegendProgressBar>
+      <UiCardFooter>
+        <template #left>
+          <p>{{ $t("vcpus-used") }}</p>
+          <p class="footer-value">{{ nVCpuInUse }}</p></template
+        >
```

Comment:
Not related to this PR: I think this kind of formatting is to prevent whitespaces? But from my understanding, those whitespaces get collapsed when the HTML is rendered so maybe we could disable it (if it's even possible) because it makes the code quite hard to read, IMO.
/cc @MathieuRA @ByScripts @Rajaa-BARHTAOUI